haskell-language-server.cabal

```diff
@@ -196,94 +189,94 @@ common example-plugins
                  Ide.Plugin.Example2
 
 common class
-  if flag(class) || flag(all-plugins)
+  if flag(class) && (impl(ghc < 9.0.1) || flag(force-plugins))
```

Comment:
Also, it might be neat to put a comment on each of these linking to the issue that describes the blocker, so if keen people end up here they know what to look for!

Reply:
ya, i also miss that in the actual `cabal-ghc<version>.project` so it will improve things

Reply:
Although all of them could point to the general issues about ghc support: https://github.com/haskell/haskell-language-server/issues/297 and https://github.com/haskell/haskell-language-server/issues/2179?